study.md

```diff
@@ -27,17 +27,19 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+DOM is not a language and DOM is not a file.
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+No, for what I understood, DOM is everything written in HTML and can be touched
+by Javascript, to change or create HTML elements on the browser of the user.
 ```
```

Comment:
The DOM does allow this type of interactivity on our webpages! This is a tricky one but the answer is kind of. What we see in Dev Tools is a visual representation of the DOM.